builder.py

```diff
@@ -51,4 +51,12 @@ def build_wheel(wheel_directory, config_settings=None, metadata_directory=None):
         except ImportError:
             sys.exit("Wayland backend requested but pywlroots is not installed.")
 
+    # Write library paths to file, if they are specified at build time via
+    # --config-settings=PANGO_PATH=...
+    with open("libqtile/build_config.py", "w") as f:
+        f.write("# This file is generated at build time by builder.py\n")
+        for lib in ["PANGO", "PANGOCAIRO", "GOBJECT", "XCBCURSOR"]:
+            p = lib + "_PATH"
+            f.write(f"{p} = {config_settings.get(p)}\n")
```

Comment:
Here the config path need to be a string, see https://github.com/qtile/qtile/pull/5420/commits/53b4e84c4b285b6884f147d7274dc8d79025f146

Reply:
Yep, thank you! I cherry picked your patch to this branch.